### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by Kugelsicher**

### CODE REVIEW

**Review by TrisTenuto — APPROVED** [review]

File: Logic/MovieLogic.cs [hunk 1]
```diff
@@ -16,44 +17,586 @@ public MovieLogic(RepoLogic repo)
         {
             _repo = repo;
         }
-        public async Task<bool> CreateMovie(string movieId)
+
+        public async Task<MovieDTO> GetMovie(string movieId)
```
Comment (markmoorerev) [inline Logic/MovieLogic.cs:21]: comments in this whole file

File: Model/TaggingDTO.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace Model
+{
+    public sealed class TaggingDTO
```
Comment (markmoorerev) [inline Model/TaggingDTO.cs:6]: purpose

File: Repository/Models/Actor.cs [hunk 3]
```diff
@@ -7,7 +7,14 @@ namespace Repository.Models
 {
     public partial class Actor
```
Comment (markmoorerev) [inline Repository/Models/Actor.cs:8]: purpose

File: Repository/Models/Director.cs [hunk 4]
```diff
@@ -7,7 +7,14 @@ namespace Repository.Models
 {
     public partial class Director
```
Comment (markmoorerev) [inline Repository/Models/Director.cs:8]: porpose

File: Repository/Models/FollowingMovie.cs [hunk 5]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+
+#nullable disable
+
+namespace Repository.Models
+{
+    public partial class FollowingMovie
```
Comment (markmoorerev) [inline Repository/Models/FollowingMovie.cs:8]: porpose

File: Repository/Models/Genre.cs [hunk 6]
```diff
@@ -7,7 +7,14 @@ namespace Repository.Models
 {
     public partial class Genre
```
Comment (markmoorerev) [inline Repository/Models/Genre.cs:8]: purpose

File: Logic/MovieLogic.cs (hunk 1 above)
Comment (Kugelsicher) [reply Logic/MovieLogic.cs:21]: The methods without comments are implementations of an interface that is commented

**Review by markmoorerev — CHANGES_REQUESTED** [review]
put comments on all the classes and methods. I didn't look at the final 14 files

File: Repository/Models/Actor.cs (hunk 3 above)
Comment (Kugelsicher) [reply Repository/Models/Actor.cs:8]: That is a database model, created by the database-first scaffolding tool.

File: Repository/Models/Director.cs (hunk 4 above)
Comment (Kugelsicher) [reply Repository/Models/Director.cs:8]: That is a database model, created by the database-first scaffolding tool.

File: Repository/Models/FollowingMovie.cs (hunk 5 above)
Comment (Kugelsicher) [reply Repository/Models/FollowingMovie.cs:8]: That is a database model, created by the database-first scaffolding tool.

File: Repository/Models/Genre.cs (hunk 6 above)
Comment (Kugelsicher) [reply Repository/Models/Genre.cs:8]: That is a database model, created by the database-first scaffolding tool.

**Review by markmoorerev — APPROVED** [review]

**Pull request merged**